torch/csrc/jit/serialization/export_bytecode.cpp

```diff
@@ -377,6 +379,9 @@ mobile::Module jitModuleToMobile(
       backend_debug_info_map.begin(), backend_debug_info_map.end());
   m.setDebugTable(MobileDebugTable(
       debug_handle_cs_ptr_map.begin(), debug_handle_cs_ptr_map.end()));
+
+  m.set_min_operator_version(get_min_operator_version_via_bytecode(m));
+  m.set_bytecode_version(options.model_version);
```

Comment:
The operator version is set when exporting the model, and it's not only for bytecode model. We probably don't need to set it here? cc: @tugsbayasgalan